types/react-navigation/index.d.ts

```diff
@@ -8,6 +8,7 @@
 //                 Kyle Roach <https://github.com/iRoachie>
 //                 phanalpha <https://github.com/phanalpha>
 //                 charlesfamu <https://github.com/charlesfamu>
+//                 Tim Wang <https://github.com/timwangdev>
```

Comment:
Adding your name here means you agree to be a maintainer. You'll receive notifications of all issues, PRs. Just making sure you know what you're doing @timwangdev